sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -33,5 +33,6 @@
     <Compile Include="Shared\DiagnosticScopeFactory.cs" />
     <Compile Include="Shared\DiagnosticScope.cs" />
     <Compile Include="Shared\TaskExtensions.cs" />
+    <Compile Include="Shared\ValueStopwatch.cs" />
```

Comment:
Don't need to include into the core itself, it's not being used there.

Reply:
To be able to test include shared source file into the test project directly (you'll see examples in the test.csproj)

Reply:
Addressed *fingerscrossed*